src/Network/DogStatsd.hs

```diff
@@ -0,0 +1,53 @@
+module Network.DogStatsd (
+ dogStatsdClient
+, increment
+, decrement
+, count
+, gauge
+, timing
+, histogram
+) where
+
+import Network.Statsd (Stat, Type(..), fmtDatagram)
+import Network.Statsd.UdpClient(UdpClient(..), fromURI, send)
+
+import Control.Monad
+import Data.Maybe
+import Data.List
+import Data.Time.Units
+import Text.Printf
+import Network.URI
+
+type Tags = [(String, String)]
+
+dogStatsdClient :: String -> IO UdpClient
+dogStatsdClient url = (fromURI . fromJust . parseURI) url
```

Comment:
This one I could use some advice on. From playing in the repl, you will get an exception from `fromJust`.